src/core/public/chrome/ui/header/header_logo.test.tsx

```diff
@@ -97,8 +97,36 @@ describe('Header logo', () => {
       expect(props.navigateToApp).toHaveBeenCalledWith('home');
     });
 
+    it('reloads the page when reload is specified to true', () => {
+      const props = {
+        ...mockProps(),
+        navigateToApp: jest.fn(),
+        reload: true, // Set reload to true
+      };
+      const component = mountWithIntl(<HeaderLogo {...props} />);
+      component.find('.logoContainer img').simulate('click');
+
+      // Expect that navigateToApp is not called when reload is true
+      expect(props.navigateToApp).not.toHaveBeenCalled();
+      // Expect that window.location.reload is called exactly once
+      expect(props.navigateToApp).toHaveBeenCalledTimes(1);
```

Comment:
Aren't these two conflicting?

Reply:
removed .not.toHaveBeenCalled()

Thanks